ext_libs/Makefile

```diff
@@ -30,7 +36,9 @@ libswresample.a: swresample-vgmstream-3.dll swresample-vgmstream-3.def
 	$(DLLTOOL) -D swresample-vgmstream-3.dll -d swresample-vgmstream-3.def -l libswresample.a
 
 libatrac9.a: libatrac9.dll libatrac9.def
-	$(DLLTOOL) -D libatrac9.dll -d libatrac9.ef -l libatrac9.a
+	$(DLLTOOL) -d libatrac9.dll -d libatrac9.def -l libatrac9.a
```

Comment:
Shouldn't that first switch be a capital D?

Reply:
Why yes, it should (leftover from a prev change; I think -D isn't needed if the .dll name is included in the .def though)